mzLib/UsefulProteomicsDatabases/ProteinDbWriter.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Proteomics;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Linq;
+using System.Globalization;
+
+namespace UsefulProteomicsDatabases
+{
+    public class ProteinDbWriter
+    {
+        #region Public Methods
+
+        public static void WriteXmlDatabase(Dictionary<string, HashSet<Tuple<int, ModificationWithMass>>> Mods, List<Protein> proteinList, string outputFileName)
+        {
+            var xmlWriterSettings = new XmlWriterSettings
+            {
+                Indent = true,
+                IndentChars = "  "
+            };
```

Comment:
Included in MetaMorpheus GPTMD code. This initialization makes the XML more human interpretable by indenting the tree structure.